README.md

```diff
@@ -5,7 +5,7 @@ the modern app server for PHP.
 
 ## Getting Started
 
-```
+```shell
```

Comment:
```suggestion
```console
```

(Nitpicking)

Reply:
I did not know this, is this documented somewhere (on Github)?

Reply:
I don't know but this allow to have the text retuned by the command on another color.